Neos.Flow/composer.json

```diff
@@ -60,7 +60,8 @@
     "require-dev": {
         "vimeo/psalm": "~4.30.0",
         "mikey179/vfsstream": "^1.6.10",
-        "phpunit/phpunit": "~9.1"
+        "phpunit/phpunit": "~9.1",
+        "behat/behat": "^3.5"
```

Comment:
Why do we need `behat/behat` in Neos.Flow AND in the development-collection? From my POV we just need it in the  development-collection and just put the tests in Neos.Flow

Reply:
ill revert that ;)

Reply:
fixed